### ISSUE TITLE
MySQL example probably should use a different image

### ISSUE THREAD

**Issue by khalwat** [craftcms/docker#27]
The MySQL example here: 

https://github.com/craftcms/docker#running-locally-with-docker-compose

```yaml
  mysql:
    image: mysql:8.0
    ports:
      - 3306:3306
    environment:
      MYSQL_ROOT_PASSWORD: SuperPassword123456!
      MYSQL_DATABASE: dev_craftcms
      MYSQL_USER: craftcms
      MYSQL_PASSWORD: SecretPassword
    volumes:
      - db_data:/var/lib/mysql
```

...probably shouldn't use the "official" image... because it doesn't work with Apple Silicon M1 processors. 

I'd recommend instead [mysql/mysql-server:8.0](https://github.com/nystudio107/plugindev/blob/master/docker-config/mysql/Dockerfile) which is the official image from Oracle. It's also a little bit smaller than the official image, but with no apparent resolution in sight:

https://stackoverflow.com/questions/44854843/docker-is-there-any-difference-between-the-two-mysql-docker-images
https://github.com/docker-library/mysql/issues/318

It's probably the way to go. DockerHub link: https://hub.docker.com/r/mysql/mysql-server

**Comment by jonathanmelville** [comment]
Thanks @khalwat, just ran into this. The `mysql/mysql-server` image does resolve it.

**Comment by timkelty** [comment]
Fixed in the latest version of the images (`1.3.1`)!

I just skirted around the issue by showing postgres in the example. Soon enough we'll add more examples and we'll include examples with `mysql/mysql-server:8.0`.
